src/main/java/in/balendin/schoolmanagment/pages/AddOrganizationPage.java

```diff
@@ -0,0 +1,185 @@
+
+import in.balendin.schoolmanagment.models.OrganizationData;
+import net.serenitybdd.core.pages.WebElementFacade;
+import net.thucydides.core.pages.PageObject;
+import org.junit.Assert;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.support.FindBy;
+
+public class AddOrganizationPage extends PageObject {
+
+    @FindBy(xpath = "/html/body/div/aside/div/nav/ul/li[1]/ul/li[3]/a")
+    private WebElementFacade subTab_Organization;
+
+    @FindBy(xpath = "/html/body/div/div[1]/section[2]/div/div/div[1]/div/a")
+    private WebElementFacade btn_AddOrganization;
+
+    // Form Elements
+    @FindBy(id = "Name")
+    private WebElementFacade organization_Name;
+
+    @FindBy(id = "Description")
+    private WebElementFacade organization_Description;
+
+    @FindBy(id = "File")
+    private WebElementFacade organization_Logo;
+
+    @FindBy(id = "Address_line1")
+    private WebElementFacade Address_line1;
+
+    @FindBy(id = "Address_line2")
+    private WebElementFacade Address_line2;
+
+    @FindBy(id = "Location")
+    private WebElementFacade Location;
+
+    @FindBy(id = "City")
+    private WebElementFacade City;
+
+    @FindBy(id = "PostalCode")
+    private WebElementFacade postal_Code;
+
+    @FindBy(id = "ContactName")
+    private WebElementFacade contact_Name;
+
+    @FindBy(id = "ContactNumber")
+    private WebElementFacade contact_Number;
+
+    @FindBy(xpath = "//*[@id=\"organizationForm\"]/div[2]/input")
+    private WebElementFacade btn_Submit;
+
+    // Popup elements
+
+    @FindBy(xpath = "/html/body/div[2]/div/div[3]/button[1]")
+    private WebElementFacade btn_OK;
+
+    // Validation messages elements
+    @FindBy(id = "Name-error")
+    private WebElementFacade name_Validation;
+
+    @FindBy(id = "Address_line1-error")
+    private WebElementFacade address1_Validation;
+
+    @FindBy(id = "Location-error")
+    private WebElementFacade loaction_Validation;
+
+    @FindBy(id = "City-error")
+    private WebElementFacade city_Validation;
+
+    @FindBy(id = "ContactName-error")
+    private WebElementFacade contactName_Validation;
+
+    @FindBy(id = "ContactNumber-error")
+    private WebElementFacade contactNumber_Validation;
+
+    @FindBy(id = "PostalCode-error")
+    private WebElementFacade postalCode_Validation;
+
+    // Sort and Search elements
+
+    @FindBy(xpath = "//*[@id=\"OrganizationList_filter\"]/label/input")
+    private WebElementFacade search_Field;
+
+    @FindBy(xpath = "//tr/th[3]")
+    private WebElementFacade organization_header;
+
+    @FindBy(xpath = "//tr/th[1]")
+    private WebElementFacade srNo_header;
+
+    @FindBy(xpath = "//*[@id=\"OrganizationList_length\"]/label/select")
+    private WebElementFacade showEntries;
+
+    @FindBy(xpath = "//*[@id=\"OrganizationList_paginate\"]/ul/li[3]/a")
+    private WebElementFacade Pagination_2;
+
+    private OrganizationData data;
+
+
+    public void navigateToAddOrganizationForm() {
+        //  tab_Admin.click();
```

Comment:
@rajaYendluri please remove the commented code

Reply:
completed